packages/core/contracts/Escrow.sol

```diff
@@ -268,24 +305,26 @@ contract Escrow is IEscrow, ReentrancyGuard {
 
         for (uint256 i = 0; i < _recipients.length; ++i) {
             if (finalAmounts[i] > 0) {
-                _safeTransfer(_recipients[i], finalAmounts[i]);
+                _safeTransfer(token, _recipients[i], finalAmounts[i]);
+                emit Payout(_recipients[i], finalAmounts[i]);
```

Comment:
I think this payout event is not required, since we already have BulkTransfer which already contains this info

Reply:
We still need `Payout` event for oracle payments to keep track of the escrow balance. Since we can't drop this event for that, I'd better use this event to make it easier to handle balance on subgraph.